.github/CONTRIBUTING.md

```diff
@@ -169,12 +169,32 @@ See [Contributing to documentation related to PowerShell](#contributing-to-docum
 
 #### Pull Request - Code Review
 
-* After a successful test pass,
-  the area maintainers will do a code review,
+* Author, Reviewer, and Assignee of a PR
+  * Reviewer and Assignee are two separate roles of a PR.
+    * A Reviewer can be anyone from the collaborators.
```

Comment:
This is a little misleading to me. "collaborators" are external people who have special status with the project (Charles, Ilya, Jeff, etc.). Do you mean that anyone can be a reviewer?

On a related note, should we specify that at least one PowerShell team member or a Collaborator must sign off for a review to be Approved?